tests/Integration/Behaviour/Features/Context/Domain/OrderFeatureContext.php

```diff
@@ -674,6 +689,14 @@ private function updateProductInOrder(int $orderId, array $productOrderDetail, a
         }
     }
 
+    /**
+     * @Then I should get no order error
```

Comment:
This could probably be useful in the CommonContext no? (without the order part)

Reply:
Fixed